index.html

```diff
@@ -1,10 +1,132 @@
-<!DOCTYPE html>
+﻿<!DOCTYPE html>
 <html lang="ru">
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Седона</title>
   </head>
   <body>
+      <header>
+          <nav>
+              <a>
+                  <img src="img/index-logo.svg" width="150" height="153"
```

Comment:
У всех тегов `img` должна быть закрывающая скобка, атрибут `alt`(для экранных читалок и на случай если изображение не загрузится) и `src` с путем к картинке. Следует перепроверить все изображения
```suggestion
                  <img src="img/index-logo.svg" width="150" height="153" alt="Седона">
```


Хороший способ понять что писать в `alt` - представить человека с нарушениями зрения, который должен по описанию понять, что изображено на картинке. [Справка с портала Веблайнд](https://weblind.ru/inner.html#image-content), [Справка с Хабра](https://habr.com/ru/post/341810/)